_benchmark/user-guide/install-and-configure/configuring-benchmark.md

```diff
@@ -159,6 +160,46 @@ This section defines how OpenSearch versions are distributed.
 | :---- | :---- | :---- |
 | `release.cache` | Boolean | Determines whether newly released OpenSearch versions should be cached locally. |
 
+## Running OpenSearch Benchmark with AWS Signature Version 4
+
+OpenSearch Benchmark supports AWS Signature Version 4 authentication. To run OpenSearch Benchmark with AWS Signature Version 4, use the following steps:
+
+1. Set up an [AWS Identity and Access Management (IAM) user or role](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create.html) and provide it access to the OpenSearch cluster using AWS Signature Version 4 authentication.
+
+2. Set up the following environment variables for your IAM user:
+
+   ```bash
+   export OSB_AWS_ACCESS_KEY_ID=<IAM USER AWS ACCESS KEY ID>
+   export OSB_AWS_SECRET_ACCESS_KEY=<IAM USER AWS SECRET ACCESS KEY>
+   export OSB_REGION=<YOUR REGION>
+   export OSB_SERVICE=es
+   ```
+   {% include copy.html %}
+
+   If you want to set up an IAM role instead of an IAM user, use the following environment variables instead:
```

Comment:
I added a link to the AWS documentation with the definition here. 

```suggestion
   If you want to set up an [IAM role](https://docs.aws.amazon.com/IAM/latest/UserGuide/id.html) instead of an IAM user, use the following environment variables instead:
```

Reply:
@natebower: Let me know if this works.

Reply:
@Naarcha-AWS I think the addition of the link is helpful, but this is still missing some context. Why would one choose to set up a user versus a role? Let's add that to step 1 as a second sentence, as applicable. Additionally, I was more thinking the following for 2 and 3:

2. For an IAM user, configure the following environment variables:
3. For an IAM role, configure the following environment variables:

Reply:
In other words, instead of steps, I would do something like this:

"OpenSearch Benchmark supports AWS Signature Version 4 authentication. To run OpenSearch Benchmark with AWS Signature Version 4, you need to set up an [AWS Identity and Access Management (IAM) user or role](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create.html) and provide it access to the OpenSearch cluster using AWS Signature Version 4 authentication. [Insert sentence about why you would use one or the other here, as applicable.]

For an IAM user, configure the following environment variables:

For an IAM role, configure the following environment variables: